pkg/yang/parse.go

```diff
@@ -185,19 +174,18 @@ Loop:
 		case nil:
 			break Loop
 		case p.hitBrace:
-			fmt.Fprintf(p.errout, "%s:%d:%d: unexpected %c\n", ns.file, ns.line, ns.col, closeBrace)
+			fmt.Fprintf(p.errout, "%s:%d:%d: unexpected %c\n", ns.file, ns.line, ns.col, '}')
 		default:
 			statements = append(statements, ns)
 		}
 	}
 
-	p.checkStatementDepth()
+	p.checkStatementDepthIsZero()
```

Comment:
Do we need `check` in the name here? `statementDepthIsZero`?

Reply:
I think the `check` is descriptive because it can output an error if the check fails.